scheduler/src/test/scala/com/sky/kms/base/SchedulerIntSpecBase.scala

```diff
@@ -2,7 +2,9 @@ package com.sky.kms.base
 
 import cats.data.NonEmptyList
 import com.sky.kms.utils.TestConfig
+import scala.concurrent.duration._
 
 abstract class SchedulerIntSpecBase extends AkkaKafkaSpecBase {
   implicit val conf = TestConfig(NonEmptyList.of(scheduleTopic, extraScheduleTopic))
+  val Tolerance     = 1300 milliseconds
```

Comment:
```suggestion
  val tolerance     = 1300 milliseconds
```

Scala [naming convention](https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods) is that only constants defined in objects should be in upper camel case

Reply:
will update